FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_Sockets.c

```diff
@@ -1356,7 +1356,8 @@ BaseType_t xReturn;
 			( lOptionName == FREERTOS_SO_SNDBUF ) ? "SND" : "RCV" ) );
 		xReturn = -pdFREERTOS_ERRNO_EINVAL;
 	}
-	else if( ( ( lOptionName == FREERTOS_SO_SNDBUF ) && ( pxSocket->u.xTCP.txStream != NULL ) ) ||
+	else
+	if( ( ( lOptionName == FREERTOS_SO_SNDBUF ) && ( pxSocket->u.xTCP.txStream != NULL ) ) ||
```

Comment:
This was better and more readable/maintainable before, I was saying that you should have `else { if( ...`.  It was fine as is given that MISRA C does allow `else if` as long as the statement ends with an `else`.  Let's put this back the way it was before.

Reply:
Oh I see! Done!